msgid ""
msgstr ""
"Project-Id-Version: duplicity\n"
"Report-Msgid-Bugs-To: Kenneth Loafman <kenneth@loafman.com>\n"
"POT-Creation-Date: 2024-11-17 10:47-0600\n"
"PO-Revision-Date: 2024-11-17 17:08\n"
"Last-Translator: \n"
"Language-Team: German, Austria\n"
"Language: de_AT\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Crowdin-Project: duplicity\n"
"X-Crowdin-Project-ID: 539508\n"
"X-Crowdin-Language: de-AT\n"
"X-Crowdin-File: /main/po/duplicity.pot\n"
"X-Crowdin-File-ID: 6\n"

#: ../duplicity/__main__.py:76
msgid "INT intercepted...exiting."
msgstr "INT abgefangen … wird beendet."

#: ../duplicity/__main__.py:84
#, python-format
msgid "GPG error detail: %s"
msgstr "GPG-Fehlerdetail: %s"

#: ../duplicity/__main__.py:91
#, python-format
msgid "User error detail: %s"
msgstr "Benutzerfehlerdetail: %s"

#: ../duplicity/__main__.py:98
#, python-format
msgid "Backend error detail: %s"
msgstr "Fehlerdetail des Hintergrundprogramms: %s"

#: ../duplicity/argparse311.py:291
msgid "usage: "
msgstr "nutzung: "

#: ../duplicity/argparse311.py:761
#, python-format
msgid "argument %(argument_name)s: %(message)s"
msgstr "Argument %(argument_name)s: %(message)s"

#: ../duplicity/argparse311.py:870
msgid ".__call__() not defined"
msgstr ".__call__() nicht definiert"

#: ../duplicity/argparse311.py:1115
#, python-format
msgid "conflicting subparser: %s"
msgstr "widersprüchlicher Subparser: %s"

#: ../duplicity/argparse311.py:1118
#, python-format
msgid "conflicting subparser alias: %s"
msgstr "conflicting subparser alias: %s"

#: ../duplicity/argparse311.py:1152
#, python-format
msgid "unknown parser %(parser_name)r (choices: %(choices)s)"
msgstr "unbekannte Parser- %(parser_name)r (Auswahl: %(choices)s)"

#: ../duplicity/argparse311.py:1215
#, python-format
msgid "argument \"-\" with mode %r"
msgstr "Argument \"-\" mit Modus %r"

#: ../duplicity/argparse311.py:1223
#, python-format
msgid "can't open '%(filename)s': %(error)s"
msgstr "kann '%(filename)s': %(error)s nicht öffnen"

#: ../duplicity/argparse311.py:1427
#, python-format
msgid "cannot merge actions - two groups are named %r"
msgstr "kann keine Aktionen zusammenführen - zwei Gruppen sind %r"

#: ../duplicity/argparse311.py:1462
msgid "'required' is an invalid argument for positionals"
msgstr "'required' ist ein ungültiges Argument für Positionale"

#: ../duplicity/argparse311.py:1483
#, python-format
msgid "invalid option string %(option)r: must start with a character %(prefix_chars)r"
msgstr "ungültiger Optionsstring %(option)r: muss mit einem Zeichen %(prefix_chars)r beginnen"

#: ../duplicity/argparse311.py:1500
#, python-format
msgid "dest= is required for options like %r"
msgstr "dest= ist erforderlich für Optionen wie %r"

#: ../duplicity/argparse311.py:1517
#, python-format
msgid "invalid conflict_resolution value: %r"
msgstr "ungültiger Konflikt_Auflösung Wert: %r"

#: ../duplicity/argparse311.py:1534
#, python-format
msgid "conflicting option string: %s"
msgid_plural "conflicting option strings: %s"
msgstr[0] "widersprüchlicher Optionszeichenkette: %s"
msgstr[1] "widersprüchliche Optionszeichen: %s"

#: ../duplicity/argparse311.py:1595
msgid "mutually exclusive arguments must be optional"
msgstr "gegenseitig ausschließende Argumente müssen optional sein"

#: ../duplicity/argparse311.py:1670
msgid "positional arguments"
msgstr "positionsargumente"

#. Used in usage help. (Should be consistent with the "Options:"
#. header.) Example:
#. duplicity [full|incremental] [options] source_path target_url
#: ../duplicity/argparse311.py:1671 ../duplicity/cli_data.py:364
#: ../duplicity/cli_data.py:529 ../duplicity/cli_data.py:583
#: ../duplicity/cli_data.py:718 ../duplicity/cli_data.py:918
msgid "options"
msgstr "Optionen"

#: ../duplicity/argparse311.py:1689
msgid "show this help message and exit"
msgstr "diese Hilfe-Nachricht anzeigen und beenden"

#: ../duplicity/argparse311.py:1721
msgid "cannot have multiple subparser arguments"
msgstr "kann nicht mehrere Subparser-Argumente haben"

#: ../duplicity/argparse311.py:1769 ../duplicity/argparse311.py:2264
#, python-format
msgid "unrecognized arguments: %s"
msgstr "nicht erkannte Argumente: %s"

#: ../duplicity/argparse311.py:1868
#, python-format
msgid "not allowed with argument %s"
msgstr "mit dem Argument %s nicht erlaubt"

#: ../duplicity/argparse311.py:1912 ../duplicity/argparse311.py:1926
#, python-format
msgid "ignored explicit argument %r"
msgstr "das explizite Argument %r ignoriert"

#: ../duplicity/argparse311.py:2030
#, python-format
msgid "the following arguments are required: %s"
msgstr "folgende Argumente werden benötigt: %s"

#: ../duplicity/argparse311.py:2042
#, python-format
msgid "one of the arguments %s is required"
msgstr "eines der Argumente %s ist erforderlich"

#: ../duplicity/argparse311.py:2083
msgid "expected one argument"
msgstr "ein Argument erwartet"

#: ../duplicity/argparse311.py:2084
msgid "expected at most one argument"
msgstr "höchstens ein Argument erwartet"

#: ../duplicity/argparse311.py:2085
msgid "expected at least one argument"
msgstr "mindestens ein Argument erwartet"

#: ../duplicity/argparse311.py:2089
#, python-format
msgid "expected %s argument"
msgid_plural "expected %s arguments"
msgstr[0] "erwartetes %s Argument"
msgstr[1] "erwartete %s Argumente"

#: ../duplicity/argparse311.py:2143
#, python-format
msgid "ambiguous option: %(option)s could match %(matches)s"
msgstr "mehrdeutige Option: %(option)s könnte %(matches)s entsprechen"

#: ../duplicity/argparse311.py:2207
#, python-format
msgid "unexpected option string: %s"
msgstr "unerwartete Optionszeichenkette: %s"

#: ../duplicity/argparse311.py:2402
#, python-format
msgid "%r is not callable"
msgstr "%r ist nicht aufrufbar"

#: ../duplicity/argparse311.py:2419
#, python-format
msgid "invalid %(type)s value: %(value)r"
msgstr "ungültiger %(type)s Wert: %(value)r"

#: ../duplicity/argparse311.py:2429
#, python-format
msgid "invalid choice: %(value)r (choose from %(choices)s)"
msgstr "ungültige Auswahl: %(value)r (Auswahl aus %(choices)s)"

#: ../duplicity/argparse311.py:2505
#, python-format
msgid "%(prog)s: error: %(message)s\n"
msgstr "%(prog)s: Fehler: %(message)s\n"

#: ../duplicity/backend.py:95
#, python-brace-format
msgid "Import of {imp} Failed"
msgstr ""

#: ../duplicity/backend.py:200
#, python-format
msgid "Could not initialize backend: %s"
msgstr "Hintergrundprogramm konnte nicht initialisiert werden: %s"

#: ../duplicity/backend.py:391
#, python-format
msgid "Backtrace of previous error: %s"
msgstr "Rückverfolgung des vorangegangenen Fehlers: %s"

#: ../duplicity/backend.py:412
#, python-format
msgid "Giving up after %s attempts. %s: %s. (for trace back: set log level DEBUG)"
msgstr ""

#: ../duplicity/backend.py:421
#, python-format
msgid "Giving up after %s attempts. %s: %s"
msgstr "Nach %s Versuchen wird aufgegeben. %s: %s"

#: ../duplicity/backend.py:427
#, python-format
msgid "Attempt of %s Nr. %s failed. %s: %s"
msgstr "Versuch %s Nr. %s fehlgeschlagen. %s: %s"

#: ../duplicity/backend.py:526
#, python-format
msgid "Reading results of '%s'"
msgstr "Ergebnisse von »%s« werden gelesen"

#: ../duplicity/backend.py:553
#, python-format
msgid "Writing %s"
msgstr "%s wird geschrieben"

#: ../duplicity/backend.py:608
#, python-format
msgid "File %s not found locally after get from backend"
msgstr "Datei %s konnte nicht lokal gefunden werden, nach Erhalt vom Hintergrundprogramm."

#: ../duplicity/backend.py:661
#, python-format
msgid "%s Remote filesize %d for %s does not match local size %d"
msgstr ""

#: ../duplicity/backends/_testbackend.py:219
#, python-format
msgid "Unexpected exception while validate %s."
msgstr ""

#: ../duplicity/backends/giobackend.py:108
#, python-format
msgid "Connection failed, please check your password: %s"
msgstr "Verbindung fehlgeschlagen, bitte Passwort prüfen: %s"

#: ../duplicity/backends/multibackend.py:100
#, python-format
msgid "MultiBackend: Could not parse query string %s: %s "
msgstr "MultiBackend: Konnte den angefragten String %s nicht auswerten: %s "

#: ../duplicity/backends/multibackend.py:110
#, python-format
msgid "MultiBackend: Invalid query string %s: more than one value for %s"
msgstr "MultiBackend: angefragter String %s ist nicht gültig: mehr als einen Wert für %s"

#: ../duplicity/backends/multibackend.py:116
#, python-format
msgid "MultiBackend: Invalid query string %s: unknown parameter %s"
msgstr "MultiBackend: angefragter String %s ist nicht gültig: unbekannter Parameter %s"

#: ../duplicity/backends/multibackend.py:167
#: ../duplicity/backends/multibackend.py:174
#, python-format
msgid "MultiBackend: illegal value for %s: %s"
msgstr "MultiBackend: nicht erlaubter Wert für %s: %s"

#: ../duplicity/backends/multibackend.py:186
#, python-format
msgid "MultiBackend: Url %s"
msgstr "MehrfachHintergrundprogramm: Adresse %s"

#: ../duplicity/backends/multibackend.py:189
#, python-format
msgid "MultiBackend: Could not load config file %s: %s "
msgstr "MehrfachHintergrundprogramm: Konfigurationsdatei konnte nicht geladen werden %s: %s "

#: ../duplicity/backends/multibackend.py:196
#, python-format
msgid "MultiBackend: use store %s"
msgstr "MehrfachHintergrundprogramm: Speicher %s benutzen"

#: ../duplicity/backends/multibackend.py:200
#, python-format
msgid "MultiBackend: set env %s = %s"
msgstr "MehrfachHintergrundprogramm: env %s = %s einstellen"

#: ../duplicity/backends/multibackend.py:214
#, python-format
msgid "Multibackend: register affinity for prefix %s"
msgstr "Multibackend: Registeraffinität für Präfix %s"

#: ../duplicity/backends/multibackend.py:257
#, python-format
msgid "MultiBackend: _put: write to store #%s (%s)"
msgstr "MehrfachHintergrundprogramm: _put: Zum Speicher schreiben #%s (%s)"

#: ../duplicity/backends/multibackend.py:272
#, python-format
msgid "MultiBackend: failed to write to store #%s (%s), try #%s, Exception: %s"
msgstr "MehrfachHintergrundprogramm: Es konnte nicht zum Speicher geschrieben werden #%s (%s), bitte #%s versuchen, Ausnahmefall: %s"

#: ../duplicity/backends/multibackend.py:286
#, python-format
msgid "MultiBackend: failed to write %s. Aborting process."
msgstr "MehrfachHintergrundprogramm: Schreiben von %s ist fehlgeschlagen. Vorgang wird abgebrochen."

#: ../duplicity/backends/multibackend.py:294
#, python-format
msgid "MultiBackend: failed to write %s. Tried all backing stores and none succeeded"
msgstr "MehrfachHintergrundprogramm: %s konnte nicht geschrieben werden. Es wurden alle Hintergrundspeicher versucht, aber ohne Erfolg"

#: ../duplicity/backends/multibackend.py:315
#, python-format
msgid "MultiBackend: failed to get %s to %s from %s"
msgstr "MehrfachHintergrundprogramm: %s konnte nicht zu %s von %s gelangen"

#: ../duplicity/backends/multibackend.py:320
#, python-format
msgid "MultiBackend: failed to get %s. Tried all backing stores and none succeeded"
msgstr "MehrfachHintergrundprogramm: %s konnte nicht erhalten werden. Es wurden alle Hintergrundspeicher versucht, aber ohne Erfolg"

#: ../duplicity/backends/multibackend.py:330
#, python-format
msgid "MultiBackend: %s: %d files"
msgstr "MultiBackend: %s: %d Dateien"

#: ../duplicity/backends/multibackend.py:334
msgid "Exception during list of {s.backend.parsed_url.strip_auth()}: "
msgstr "Ausnahme während der Liste von {s.backend.parsed_url.strip_auth()}: "

#: ../duplicity/backends/multibackend.py:342
#, python-format
msgid "MultiBackend: combined list: %s"
msgstr "MehrfachHintergrundprogramm: kombinierte Liste: %s"

#: ../duplicity/backends/multibackend.py:371
#: ../duplicity/backends/multibackend.py:398
#, python-format
msgid "MultiBackend: failed to delete %s. Tried all backing stores and none succeeded"
msgstr "MehrfachHintergrundprogramm: %s konnte nicht gelöscht werden. Es wurden alle Hintergrundspeicher versucht, aber ohne Erfolg"

#: ../duplicity/backends/onedrivebackend.py:88
#, python-format
msgid "Attempt of initialize_oauth2_session Nr. %s failed. %s: %s"
msgstr "Versuch der Initialize_oauth2_session Nr. %s fehlgeschlagen. %s: %s"

#: ../duplicity/backends/pydrivebackend.py:203
#, python-format
msgid "PyDrive backend: multiple files called '%s'."
msgstr "PyDrive-Hintergrundprogramm: mehrere Dateien aufgerufen »%s«."

#: ../duplicity/backends/webdavbackend.py:67
msgid "Missing socket or ssl python modules."
msgstr "Es fehlt ein Socket oder ein SSL Python Modul."

#: ../duplicity/backends/webdavbackend.py:87
#, python-format
msgid "Cacert database file '%s' is not readable."
msgstr "Die Cacert Datenbankdatei '%s' ist nicht lesbar."

#: ../duplicity/backends/webdavbackend.py:152
#, python-format
msgid "Using WebDAV host %s port %s"
msgstr "WebDAV-Rechner %s Anschluss (Port) %s wird verwendet"

#: ../duplicity/backends/webdavbackend.py:153
#, python-format
msgid "Using WebDAV directory %s"
msgstr "WebDAV-Verzeichnis %s wird verwendet"

#: ../duplicity/backends/webdavbackend.py:183
#, python-format
msgid "WebDAV create connection on '%s'"
msgstr "WebDAV erzeugt eine Verbindung unter '%s'"

#: ../duplicity/backends/webdavbackend.py:191
#, python-format
msgid "WebDAV Unknown URI scheme: %s"
msgstr "WebDAV unbekanntes URI Schema: %s"

#: ../duplicity/backends/webdavbackend.py:210
#, python-format
msgid "WebDAV %s %s request with headers: %s "
msgstr "WebDAV %s %s Anfrage mit Headern: %s "

#: ../duplicity/backends/webdavbackend.py:211
#: ../duplicity/backends/webdavbackend.py:234
#, python-format
msgid "WebDAV data length: %s "
msgstr "WebDAV Datenlänge: %s "

#: ../duplicity/backends/webdavbackend.py:214
#, python-format
msgid "WebDAV response status %s with reason '%s'."
msgstr "WebDAV Antwortstatus %s mit dem Grund '%s'."

#: ../duplicity/backends/webdavbackend.py:220
#, python-format
msgid "WebDAV redirect to: %s "
msgstr "WebDAV weitersenden an: %s "

#: ../duplicity/backends/webdavbackend.py:222
msgid "WebDAV redirected 10 times. Giving up."
msgstr "WebDAV leitete 10 mal um. Gebe auf."

#: ../duplicity/backends/webdavbackend.py:227
msgid "WebDAV missing location header in redirect response."
msgstr "WebDAV fehlender Lokalisierungs-Header in redirect response."

#: ../duplicity/backends/webdavbackend.py:232
msgid "WebDAV retry request with authentification headers."
msgstr "WebDAV versucht erneut, eine Anforderung mit Authentifizierungs-Headern zu stellen."

#: ../duplicity/backends/webdavbackend.py:233
#, python-format
msgid "WebDAV %s %s request2 with headers: %s "
msgstr "WebDAV %s %s Anfrage2 mit Headern: %s "

#: ../duplicity/backends/webdavbackend.py:237
#, python-format
msgid "WebDAV response2 status %s with reason '%s'."
msgstr "WebDAV Antwort2 Status %s mit Grund »%s«."

#: ../duplicity/backends/webdavbackend.py:256
msgid "python-kerberos needed to use kerberos                           authorization, falling back to basic auth."
msgstr "python-kerberos benötigt Kerberos-Legitimierung, auf die Basislegitimierung wird zurückgegriffen."

#: ../duplicity/backends/webdavbackend.py:264
#, python-format
msgid "Kerberos authorization failed: %s.                          Falling back to basic auth."
msgstr "Kerberos-Legitimierung ist fehlgeschlagen: %s. Auf die Basislegitimierung wird zurückgegriffen."

#: ../duplicity/backends/webdavbackend.py:365
#, python-format
msgid "Creating missing directory %s"
msgstr "Fehlendes Verzeichnis »%s« wird erstellt"

#: ../duplicity/backends/webdavbackend.py:369
#, python-format
msgid "WebDAV MKCOL %s failed: %s %s"
msgstr "WebDAV MKCOL %s fehlgeschlagen: %s %s"

#: ../duplicity/backends/webdavbackend.py:381
#, python-format
msgid "WebDAV path decoding and translation: %s -> %s"
msgstr "WebDAV Pfad-Dekodierung und Übersetzung: %s -> %s"

#: ../duplicity/backends/webdavbackend.py:417
#, python-format
msgid "WebDAV GET Bad status code %s reason %s."
msgstr "WebDAV GET Falscher Statuscode %s Grund %s."

#: ../duplicity/backends/webdavbackend.py:438
#, python-format
msgid "WebDAV PUT Bad status code %s reason %s."
msgstr "WebDAV PUT Falscher Statuscode %s Grund %s."

#: ../duplicity/backends/webdavbackend.py:457
#, python-format
msgid "WebDAV DEL Bad status code %s reason %s."
msgstr "WebDAV DEL Falscher Statuscode %s Grund %s."

#. Used in usage help to represent a Unix-style path name. Example:
#. --archive-dir <path>
#: ../duplicity/cli_data.py:103 ../duplicity/cli_data.py:173
#: ../duplicity/cli_data.py:207 ../duplicity/cli_data.py:255
#: ../duplicity/cli_data.py:358 ../duplicity/cli_data.py:381
#: ../duplicity/cli_data.py:552 ../duplicity/cli_data.py:744
#: ../duplicity/cli_data.py:927
msgid "path"
msgstr "Pfad"

#: ../duplicity/cli_data.py:116
msgid "Hot|Cool|Archive"
msgstr "Hot|Cool|Archiv"

#. Used in usage help to represent a desired number of
#. something. Example:
#. --num-retries <number>
#: ../duplicity/cli_data.py:121 ../duplicity/cli_data.py:127
#: ../duplicity/cli_data.py:135 ../duplicity/cli_data.py:163
#: ../duplicity/cli_data.py:179 ../duplicity/cli_data.py:451
#: ../duplicity/cli_data.py:471 ../duplicity/cli_data.py:518
#: ../duplicity/cli_data.py:540 ../duplicity/cli_data.py:546
#: ../duplicity/cli_data.py:564 ../duplicity/cli_data.py:635
#: ../duplicity/cli_data.py:643 ../duplicity/cli_data.py:695
#: ../duplicity/cli_data.py:781 ../duplicity/cli_data.py:914
msgid "number"
msgstr "Zahl"

#. Used in usage help. Example:
#. --timeout <seconds>
#: ../duplicity/cli_data.py:147 ../duplicity/cli_data.py:751
#: ../duplicity/cli_data.py:940
msgid "seconds"
msgstr "Sekunden"

#. Used in usage help to represent an ID for a GnuPG key. Example:
#. --encrypt-key <gpg_key_id>
#: ../duplicity/cli_data.py:195 ../duplicity/cli_data.py:201
#: ../duplicity/cli_data.py:375 ../duplicity/cli_data.py:702
#: ../duplicity/cli_data.py:906
msgid "gpg-key-id"
msgstr "GPG-Schlüsselkennung"

#. Used in usage help to represent a "glob" style pattern for
#. matching one or more files, as described in the documentation.
#. Example:
#. --exclude <shell_pattern>
#: ../duplicity/cli_data.py:212 ../duplicity/cli_data.py:403
#: ../duplicity/cli_data.py:945
msgid "shell_pattern"
msgstr "shell_Muster"

#. Used in usage help to represent the name of a file. Example:
#. --log-file <filename>
#: ../duplicity/cli_data.py:224 ../duplicity/cli_data.py:230
#: ../duplicity/cli_data.py:291 ../duplicity/cli_data.py:409
#: ../duplicity/cli_data.py:903
msgid "filename"
msgstr "Dateiname"

#. Used in usage help to represent a time spec for a previous
#. point in time, as described in the documentation. Example:
#. duplicity remove-older-than time [options] target_url
#: ../duplicity/cli_data.py:236 ../duplicity/cli_data.py:577
#: ../duplicity/cli_data.py:971
msgid "time"
msgstr "Zeit"

#: ../duplicity/cli_data.py:249 ../duplicity/cli_data.py:415
msgid "regex"
msgstr "regex"

#: ../duplicity/cli_data.py:352
msgid "interval"
msgstr "intervall"

#: ../duplicity/cli_data.py:398
msgid "imap_mailbox"
msgstr "IMAP-Postfach"

#: ../duplicity/cli_data.py:423
msgid "If set, an extra file with statistics in json format will be stored with each backup. It includes the statistics printed out on stdout by default. With 'collection-status' it returs these stats."
msgstr "Wenn aktiviert, wird bei jedem Backup eine zusätzliche Datei mit Statistiken im json-Format gespeichert. Es enthält die Statistiken, die standardmäßig auf Stdout ausgedruckt werden. Mit dem 'Sammelstatus' gibt es diese Statistiken zurück."

#: ../duplicity/cli_data.py:431
msgid "file_descriptor"
msgstr "Dateideskriptor"

#: ../duplicity/cli_data.py:438
msgid "log_filename"
msgstr "log_Dateiname"

#: ../duplicity/cli_data.py:477
msgid "backup name"
msgstr "Sicherungsname"

#: ../duplicity/cli_data.py:594
msgid "s3_endpoint_url"
msgstr "s3_endpoint_url"

#: ../duplicity/cli_data.py:662
msgid "s3_kms_key_id"
msgstr "3_km_Schlüssel_id"

#: ../duplicity/cli_data.py:668
msgid "s3_kms_grant"
msgstr "3_km_gewähren"

#: ../duplicity/cli_data.py:674
msgid "s3_region_name"
msgstr "s3_region_name"

#: ../duplicity/cli_data.py:680
msgid "policy"
msgstr "Richtlinie"

#. noun
#: ../duplicity/cli_data.py:685 ../duplicity/cli_data.py:690
#: ../duplicity/cli_data.py:892
msgid "command"
msgstr "Befehl"

#: ../duplicity/cli_data.py:730
msgid "pem formatted bundle of certificate authorities"
msgstr "PEM-formatiertes Paket von Zertifizierungsstellen"

#: ../duplicity/cli_data.py:735
msgid "path to a folder with certificate authority files"
msgstr "Pfad zu einem Verzeichnis mit Dateien der Zertifizierungsstellen"

#. abbreviation for "character" (noun)
#: ../duplicity/cli_data.py:757 ../duplicity/cli_data.py:890
msgid "char"
msgstr "Zeichen"

#: ../duplicity/cli_data.py:770
msgid "verb"
msgstr "verb"

#: ../duplicity/cli_data.py:788
msgid "extra headers for Webdav, like 'Cookie,name=value'"
msgstr "zusätzliche Header für Webdav, wie 'Cookie,name=value'"

#. Used in usage help to represent a Unix-style path name. Example:
#. rsync://user[:password]@other_host[:port]//absolute_path
#: ../duplicity/cli_data.py:881
msgid "absolute_path"
msgstr "absoluter_Pfad"

#. Used in usage help. Example:
#. tahoe://alias/some_dir
#: ../duplicity/cli_data.py:884
msgid "alias"
msgstr "Alias"

#. Used in help to represent a "bucket name" for Amazon Web
#. Services' Simple Storage Service (S3). Example:
#. s3://other.host/bucket_name[/prefix]
#: ../duplicity/cli_data.py:888
msgid "bucket_name"
msgstr "Bucket_Name"

#. Used in usage help to represent the name of a container in
#. Amazon Web Services' Cloudfront. Example:
#. cf+http://container_name
#: ../duplicity/cli_data.py:896
msgid "container_name"
msgstr "Containername"

#. noun
#: ../duplicity/cli_data.py:898
msgid "count"
msgstr "Anzahl"

#. Used in usage help to represent the name of a file directory
#: ../duplicity/cli_data.py:900
msgid "directory"
msgstr "Verzeichnis"

#. Used in usage help, e.g. to represent the name of a code
#. module. Example:
#. rsync://user[:password]@other.host[:port]::/module/some_dir
#: ../duplicity/cli_data.py:910
msgid "module"
msgstr "Modul"

#. Used in usage help to represent an internet hostname. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/cli_data.py:921
msgid "other.host"
msgstr "anderer.Rechner"

#. Used in usage help. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/cli_data.py:924
msgid "password"
msgstr "Passwort"

#. Used in usage help to represent a TCP port number. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/cli_data.py:930
msgid "port"
msgstr "Port"

#. Used in usage help. This represents a string to be used as a
#. prefix to names for backup files created by Duplicity. Example:
#. s3://other.host/bucket_name[/prefix]
#: ../duplicity/cli_data.py:934
msgid "prefix"
msgstr "Präfix"

#. Used in usage help to represent a Unix-style path name. Example:
#. rsync://user[:password]@other.host[:port]/relative_path
#: ../duplicity/cli_data.py:937
msgid "relative_path"
msgstr "relativer_Pfad"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory. Example:
#. file:///some_dir
#: ../duplicity/cli_data.py:949
msgid "some_dir"
msgstr "irgendein_Verzeichnis"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory where files will be
#. coming FROM. Example:
#. duplicity [full|incremental] [options] source_path target_url
#: ../duplicity/cli_data.py:954
msgid "source_path"
msgstr "quelle_Pfad"

#. Used in usage help to represent a URL files will be coming
#. FROM. Example:
#. duplicity [restore] [options] source_url target_dir
#: ../duplicity/cli_data.py:958
msgid "source_url"
msgstr "Quelladresse"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory. where files will be
#. going TO. Example:
#. duplicity [restore] [options] source_url target_dir
#: ../duplicity/cli_data.py:963
msgid "target_dir"
msgstr "Zielverzeichnis"

#. Used in usage help to represent a URL files will be going TO.
#. Example:
#. duplicity [full|incremental] [options] source_path target_url
#: ../duplicity/cli_data.py:967
msgid "target_url"
msgstr "Zieladresse"

#. Used in usage help to represent a user name (i.e. login).
#. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/cli_data.py:975
msgid "user"
msgstr "Benutzer"

#. account id for b2. Example: b2://account_id@bucket/
#: ../duplicity/cli_data.py:977
msgid "account_id"
msgstr "Kontokennung"

#. application_key for b2.
#. Example: b2://account_id:application_key@bucket/
#: ../duplicity/cli_data.py:980
msgid "application_key"
msgstr "Anwendungsschlüssel"

#. remote name for rclone.
#. Example: rclone://remote:/some_dir
#: ../duplicity/cli_data.py:983 ../duplicity/dup_collections.py:565
msgid "remote"
msgstr "Entfernt"

#: ../duplicity/cli_data.py:987
msgid "Backends and their URL formats:"
msgstr "Hintergrundprogramme und deren Adressformate:"

#: ../duplicity/cli_main.py:151
msgid "No valid action found. Will imply 'backup' because a path source was given and target is a url location."
msgstr "Keine gültige Aktion gefunden. Wird 'backup' implizieren, da eine Pfadquelle angegeben wurde und das Ziel eine Url-Position ist."

#: ../duplicity/cli_main.py:159
msgid "No valid action found. Will imply 'restore' because url source was given and target is a local path."
msgstr "Keine gültige Aktion gefunden. Wird 'Wiederherstellen' implizieren, da die Url-Quelle angegeben wurde und das Ziel ein lokaler Pfad ist."

#: ../duplicity/cli_main.py:166
#, python-brace-format
msgid "Invalid '{remainder[0]}' action and cannot be implied from the "
msgstr "Ungültige Aktion '{remainder[0]}' und kann nicht von der "

#: ../duplicity/cli_main.py:266
#, python-brace-format
msgid "GPG binary is {config.gpg_binary}, version {gpg_version}"
msgstr ""

#: ../duplicity/cli_main.py:315
#, python-brace-format
msgid "Using archive dir: {config.archive_dir_path.uc_name}"
msgstr "Archivverzeichnis verwenden: {config.archive_dir_path.uc_name}"

#: ../duplicity/cli_main.py:316
#, python-brace-format
msgid "Using backup name: {config.backup_name}"
msgstr "Verwende Sicherungsname: {config.backup_name}"

#: ../duplicity/cli_util.py:45
msgid "Enter 'duplicity --help' for help screen."
msgstr "Geben Sie 'duplicity --help' für den Hilfe-Bildschirm ein."

#: ../duplicity/cli_util.py:127
msgid "Running in 'ignore errors' mode due to --ignore-errors.\n"
"Please reconsider if this was not intended"
msgstr "Im Modus 'Ignoriere Fehler' aufgrund von --ignore-errors.\n"
"Bitte überdenken, ob dies nicht beabsichtigt war"

#: ../duplicity/cli_util.py:170
msgid "'{val}' is not an int: {str(e)}"
msgstr "'{val}' ist keine Int: {str(e)}"

#: ../duplicity/cli_util.py:184
#, python-brace-format
msgid "'{val} is not a single character."
msgstr "'{val} ist kein einzelnes Zeichen."

#: ../duplicity/cli_util.py:211
#, python-brace-format
msgid "Source should be pathname, not url.  Got '{val}' instead."
msgstr "Quelle sollte Pfadname sein, nicht Url. Stattdessen '{val}' erhalten."

#: ../duplicity/cli_util.py:213
#, python-brace-format
msgid "Argument source_path '{val}' does not exist."
msgstr "Argument source_path '{val}' existiert nicht."

#: ../duplicity/cli_util.py:219
#, python-brace-format
msgid "Source should be url, not directory.  Got '{val}' instead."
msgstr "Quelle sollte url sein, nicht Verzeichnis. Stattdessen '{val}' erhalten."

#: ../duplicity/cli_util.py:225
#, python-brace-format
msgid "Target should be directory, not url.  Got '{val}' instead."
msgstr "Ziel sollte Verzeichnis, nicht URL sein. Stattdessen '{val}' erhalten."

#: ../duplicity/cli_util.py:230
msgid "Unable to create target dir '{val}': {str(e)}"
msgstr "Zielverzeichnis '{val}' konnte nicht erstellt werden: {str(e)}"

#: ../duplicity/cli_util.py:236
#, python-brace-format
msgid "Target should be url, not directory.  Got '{val}' instead."
msgstr "Ziel sollte url sein, nicht Verzeichnis. Stattdessen '{val}' erhalten."

#: ../duplicity/cli_util.py:281
msgid "Verbosity must be one of: digit [0-9], character [ewnid],\n"
"or word ['error', 'warning', 'notice', 'info', 'debug'].\n"
"The default is 3 (Notice).  It is strongly recommended\n"
"that verbosity level is set at 2 (Warning) or higher."
msgstr ""

#: ../duplicity/cli_util.py:425
#, python-brace-format
msgid "Specified archive directory '{archive_dir_path.uc_name}' is not a directory"
msgstr "Das angegebene Archivverzeichnis '{archive_dir_path.uc_name}' ist kein Verzeichnis"

#: ../duplicity/cli_util.py:434
msgid "Encrypt key should be an 8, 16, or 40 character hex string, like 'AA0E73D2'.\n"
msgstr "Schlüssel sollte ein 8, 16 oder 40 Zeichen hex String sein, wie 'AA0E73D2'.\n"

#: ../duplicity/cli_util.py:454
msgid "Hidden dncrypt key should be an 8, 16, or 40 character hex string, like 'AA0E73D2'.\n"
msgstr "Versteckte dncrypt Schlüssel sollte ein 8, 16 oder 40 Zeichen hex String sein, wie 'A0E73D2'.\n"

#: ../duplicity/cli_util.py:468
msgid "Sign key should be an 8, 16, or 40 character hex string, like 'AA0E73D2'.\n"
msgstr "Signaturschlüssel sollte ein 8, 16 oder 40 Zeichen hex String sein, wie 'AA0E73D2'.\n"

#: ../duplicity/diffdir.py:105 ../duplicity/diffdir.py:404
#, python-format
msgid "Error %s getting delta for %s"
msgstr "Fehler %s beim Erhalt des Delta für %s"

#: ../duplicity/diffdir.py:119
#, python-format
msgid "Getting delta of %s and %s"
msgstr "Delta von %s und %s wird erhalten"

#: ../duplicity/diffdir.py:163
#, python-format
msgid "A %s"
msgstr "Ein %s"

#: ../duplicity/diffdir.py:171
#, python-format
msgid "M %s"
msgstr "M %s"

#: ../duplicity/diffdir.py:194
#, python-format
msgid "Comparing %s and %s"
msgstr "%s und %s werden verglichen"

#: ../duplicity/diffdir.py:207
#, python-format
msgid "D %s"
msgstr "D %s"

#: ../duplicity/dup_collections.py:200 ../duplicity/dup_collections.py:209
#, python-format
msgid "BackupSet.delete: missing %s"
msgstr "Sicherungssatz löschen: %s fehlt"

#: ../duplicity/dup_collections.py:235
msgid "Fatal Error: No manifests found for most recent backup"
msgstr "Kritischer Fehler: Kein Manifest zur letzten Sicherung gefunden"

#: ../duplicity/dup_collections.py:248
msgid "Fatal Error: Remote manifest does not match local one.  Either the remote backup set or the local archive directory has been corrupted."
msgstr "Kritischer Fehler: Manifest im Hintergrundprogramm stimmt nicht mit der lokalen Version überein. Entweder der entfernte Sicherungssatz oder das lokale Archivverzeichnis ist fehlerhaft."

#: ../duplicity/dup_collections.py:256
#, python-format
msgid "Error processing remote manifest (%s): %s"
msgstr "Fehler beim Verarbeiten des entfernten Manifest (%s): %s"

#: ../duplicity/dup_collections.py:265
msgid "Fatal Error: Neither remote nor local manifest is readable."
msgstr "Kritischer Fehler: Weder das entfernte noch das lokale Manifest ist lesbar."

#: ../duplicity/dup_collections.py:276
#, python-format
msgid "Processing local manifest %s (%s)"
msgstr "Verarbeitung des lokalen Manifest %s (%s)"

#: ../duplicity/dup_collections.py:291
msgid "Error processing remote file ({os.fsdecode(remote_file)}): {util.uexc(message)}"
msgstr "Fehler beim Verarbeiten der entfernten Datei ({os.fsdecode(remote_file)}): {util.uexc(message)}"

#: ../duplicity/dup_collections.py:293
msgid "Processing remote file {os.fsdecode(remote_file)} ({len(remote_file_buffer)})"
msgstr "Verarbeite entfernte Datei {os.fsdecode(remote_file)} ({len(remote_file_buffer)})"

#: ../duplicity/dup_collections.py:311
msgid "No Jsonstat file found, return empty."
msgstr ""

#: ../duplicity/dup_collections.py:404
msgid "Preferring Backupset over previous one!"
msgstr "Sicherungssatz wird dem Vorherigen vorgezogen!"

#: ../duplicity/dup_collections.py:408
#, python-format
msgid "Ignoring incremental Backupset (start_time: %s; needed: %s)"
msgstr "Schrittweiser Sicherungssatz wird vernachlässigt (Startzeit: %s; benötigt: %s)"

#: ../duplicity/dup_collections.py:417
#, python-format
msgid "Added incremental Backupset (start_time: %s / end_time: %s)"
msgstr "Schrittweiser Sicherungssatz wurde hinzugefügt (Startzeit: %s; Endzeit: %s)"

#: ../duplicity/dup_collections.py:490
msgid "Chain start time: "
msgstr "Erste Sicherung in dieser Kette: "

#: ../duplicity/dup_collections.py:491
msgid "Chain end time: "
msgstr "Letzte Sicherung in dieser Kette: "

#: ../duplicity/dup_collections.py:492
#, python-format
msgid "Number of contained backup sets: %d"
msgstr "Anzahl der enthaltenen Sicherungssätze: %d"

#: ../duplicity/dup_collections.py:493
#, python-format
msgid "Total number of contained volumes: %d"
msgstr "Gesamtanzahl der enthaltenen Datenträger: %d"

#: ../duplicity/dup_collections.py:494 ../duplicity/dup_collections.py:1301
msgid "Type of backup set:"
msgstr "Typ des Sicherungssatzes:"

#: ../duplicity/dup_collections.py:494 ../duplicity/dup_collections.py:1301
msgid "Time:"
msgstr "Zeit:"

#: ../duplicity/dup_collections.py:494
msgid "Num volumes:"
msgstr "Datenträgeranz.:"

#: ../duplicity/dup_collections.py:499 ../duplicity/dup_collections.py:1308
msgid "Full"
msgstr "Vollständig"

#: ../duplicity/dup_collections.py:502 ../duplicity/dup_collections.py:1310
msgid "Incremental"
msgstr "Schrittweise"

#: ../duplicity/dup_collections.py:563
msgid "local"
msgstr "Lokal"

#: ../duplicity/dup_collections.py:726
msgid "Collection Status"
msgstr "Sammlungszustand"

#: ../duplicity/dup_collections.py:728
#, python-format
msgid "Connecting with backend: %s"
msgstr "Mit Hintergrundprogramm wird verbunden: %s"

#: ../duplicity/dup_collections.py:729
#, python-format
msgid "Archive dir: %s"
msgstr "Archivverzeichnis: %s"

#: ../duplicity/dup_collections.py:732
#, python-format
msgid "Found %d secondary backup chain(s)."
msgstr "%d sekundäre Sicherungskette(n) gefunden."

#: ../duplicity/dup_collections.py:734
#, python-format
msgid "Secondary chain %d of %d:"
msgstr "Zusätzliche Kette %d von %d:"

#: ../duplicity/dup_collections.py:739
msgid "Found primary backup chain with matching signature chain:"
msgstr "Primäre Sicherungskette mit passender Signaturkette gefunden:"

#: ../duplicity/dup_collections.py:742
msgid "No backup chains with active signatures found"
msgstr "Keine Sicherungsketten mit aktiven Signaturen gefunden"

#: ../duplicity/dup_collections.py:745
#, python-format
msgid "Also found %d backup set(s) not part of any chain,"
msgstr "Auch %d Sicherungssatz(e) gefunden, die nicht Teil einer Kette sind,"

#: ../duplicity/dup_collections.py:746
#, python-format
msgid "and %d incomplete backup set(s)."
msgstr "und %d unvollständige Backup-Set(en)."

#. "cleanup" is a hard-coded command, so do not translate it
#: ../duplicity/dup_collections.py:748
msgid "These may be deleted by running duplicity with the \"cleanup\" command."
msgstr "Sie können diese entfernen, indem Sie duplicity mit dem »cleanup«-Befehl starten."

#: ../duplicity/dup_collections.py:750
msgid "No orphaned or incomplete backup sets found."
msgstr "Keine verwaisten oder unvollständigen Sicherungssätze gefunden."

#: ../duplicity/dup_collections.py:766
#, python-format
msgid "%d file(s) exists on backend"
msgstr "%d Datei(en) existiert im Backend"

#: ../duplicity/dup_collections.py:770
#, python-format
msgid "%d file(s) exist in cache"
msgstr ""

#: ../duplicity/dup_collections.py:824
msgid "Warning, discarding last backup set, because of missing signature file."
msgstr "Achtung, der letzte Sicherungssatz wird verworfen, weil die Signaturdatei fehlt."

#: ../duplicity/dup_collections.py:847
msgid "Warning, found the following local orphaned signature file(s):"
msgstr "Warnung, folgende lokale verwaiste Signaturdatei(en) gefunden:"

#: ../duplicity/dup_collections.py:855
msgid "Warning, found the following remote orphaned signature file(s):"
msgstr "Warnung, folgende aus der Ferne verwaiste Signaturdatei(en) gefunden:"

#: ../duplicity/dup_collections.py:863
msgid "Warning, found signatures but no corresponding backup files"
msgstr "Achtung, es wurden Signaturen ohne dazugehörige Sicherungsdaten gefunden"

#: ../duplicity/dup_collections.py:869
msgid "Warning, found incomplete backup sets, probably left from aborted session"
msgstr "Achtung, es wurden unvollständige Sicherungssätze gefunden, wahrscheinlich von einer abgebrochenen Sitzung zurückgelassen"

#: ../duplicity/dup_collections.py:875
msgid "Warning, found the following orphaned backup file(s):"
msgstr "Warnung, folgende verwaiste Sicherungsdatei(en) gefunden:"

#: ../duplicity/dup_collections.py:890
#, python-format
msgid "Extracting backup chains from list of files: %s"
msgstr "Sicherungsketten aus der Liste der Dateien werden entnommen: %s"

#: ../duplicity/dup_collections.py:901
#, python-format
msgid "File %s is part of known set"
msgstr "Datei %s ist Teil eines bekannten Satzes"

#: ../duplicity/dup_collections.py:904
#, python-format
msgid "File %s is not part of a known set; creating new set"
msgstr "Datei %s ist nicht Teil eines bekannten Sicherungssatzes; neuer Satz wird erstellt"

#: ../duplicity/dup_collections.py:909
#, python-format
msgid "Ignoring file (rejected by backup set) '%s'"
msgstr "Datei »%s« wird vernachlässigt (vom Sicherungssatz abgelehnt)"

#: ../duplicity/dup_collections.py:925
#, python-format
msgid "Found backup chain %s"
msgstr "Sicherungskette %s gefunden"

#: ../duplicity/dup_collections.py:930
#, python-format
msgid "Added set %s to pre-existing chain %s"
msgstr "Satz %s zur bestehenden Sicherungskette %s hinzugefügt"

#: ../duplicity/dup_collections.py:933
#, python-format
msgid "Found orphaned set %s"
msgstr "Verwaister Satz %s gefunden"

#: ../duplicity/dup_collections.py:1081
#, python-format
msgid "No signature chain for the requested time. Using oldest available chain, starting at time %s."
msgstr "Keine Signaturkette für die angeforderte Zeit. Nutze die älteste verfügbare Kette, starte bei der Zeit %s."

#: ../duplicity/dup_collections.py:1299
#, python-format
msgid "File: %s"
msgstr "Datei: %s"

#: ../duplicity/dup_collections.py:1300
#, python-format
msgid "Total number of backup: %d"
msgstr "Gesamtzahl an Sicherungen: %d"

#: ../duplicity/dup_collections.py:1301 ../duplicity/dup_collections.py:1337
msgid "Type of file change:"
msgstr "Art der Dateiänderung:"

#: ../duplicity/dup_collections.py:1335
#, python-format
msgid " Backup set time: %s"
msgstr " Sicherungszeit: %s"

#: ../duplicity/dup_collections.py:1336
#, python-format
msgid "Total number of changes: %d"
msgstr "Gesamtzahl der Änderungen: %d"

#: ../duplicity/dup_collections.py:1337
msgid "File:"
msgstr "Datei:"

#: ../duplicity/dup_main.py:109
msgid "Reuse configured PASSPHRASE as SIGN_PASSPHRASE"
msgstr "Eingestellte PASSPHRASE als SIGN_PASSPHRASE benutzen"

#: ../duplicity/dup_main.py:120
msgid "Reuse configured SIGN_PASSPHRASE as PASSPHRASE"
msgstr "Eingestellte SIGN_PASSPHRASE als PASSPHRASE benutzen"

#: ../duplicity/dup_main.py:155
msgid "PASSPHRASE variable not set, asking user."
msgstr "Variable PASSPHRASE nicht eingestellt, Benutzer wird gefragt."

#: ../duplicity/dup_main.py:180
msgid "Retype passphrase for signing key to confirm: "
msgstr "Zur Bestätigung bitte die Passphrase für die Schlüsselsignierung wiederholen: "

#: ../duplicity/dup_main.py:182
msgid "Retype passphrase for decryption to confirm: "
msgstr "Passphrase zur Bestätigung der Entschlüsselung erneut eingeben: "

#: ../duplicity/dup_main.py:186
msgid "First and second passphrases do not match!  Please try again."
msgstr "Erste und zweite Passphrase stimmen nicht überein! Bitte nochmal versuchen."

#: ../duplicity/dup_main.py:197
msgid "Cannot use empty passphrase with symmetric encryption!  Please try again."
msgstr "Bei symmetrischer Verschlüsselung darf keine leere Passphrase benutzt werden! Bitte nochmal versuchen."

#: ../duplicity/dup_main.py:257
#, python-format
msgid "File %s complete in backup set.\n"
"Continuing restart on file %s."
msgstr "Datei %s ist komplett im Sicherungssatz.\n"
"Nach Neustart erfolgt Fortsetzung mit Datei %s."

#: ../duplicity/dup_main.py:267
#, python-format
msgid "File %s missing in backup set.\n"
"Continuing restart on file %s."
msgstr "Datei %s fehlt im Sicherungssatz.\n"
"Beim Neustart wird fortgefahren mit Datei %s."

#: ../duplicity/dup_main.py:322
#, python-format
msgid "File %s was corrupted during upload."
msgstr "Datei %s wurde während des Hochladens beschädigt."

#: ../duplicity/dup_main.py:352
msgid "Restarting backup, but current encryption settings do not match original settings"
msgstr "Datensicherung wird neu gestartet, die aktuellen Verschlüsselungseinstellungen entsprechen jedoch nicht den ursprünglichen Einstellungen"

#: ../duplicity/dup_main.py:375 ../duplicity/dup_main.py:511
#, python-format
msgid "Processed volume %d"
msgstr "Datenträger %d wurde verarbeitet"

#: ../duplicity/dup_main.py:423
msgid "Skipping encryption validation due to glacier/deep storage"
msgstr "Überspringe Verschlüsselungsüberprüfung wegen Gletscher/Tiefenspeicherung"

#: ../duplicity/dup_main.py:427
#, python-format
msgid "Restarting after volume %s, file %s, block %s"
msgstr "Neustart nach Datenträger %s, Datei %s, Block %s"

#: ../duplicity/dup_main.py:479
msgid "Skipped volume upload, as effectivly nothing has changed"
msgstr "Lautstärke-Upload übersprungen, da sich effektiv nichts geändert hat"

#: ../duplicity/dup_main.py:702
msgid "Fatal Error: Unable to start incremental backup.  Old signatures not found and incremental specified"
msgstr "Schwerwiegender Fehler: schrittweise Sicherung konnte nicht gestartet werden. Alte Signaturen nicht gefunden und schrittweise Sicherung ausgewählt."

#: ../duplicity/dup_main.py:708
msgid "No signatures found, switching to full backup."
msgstr "Keine Signaturen gefunden, vollständige Sicherung wird durchgeführt."

#: ../duplicity/dup_main.py:722
msgid "Backup Statistics"
msgstr "Sicherungsstatistiken"

#: ../duplicity/dup_main.py:862
#, python-format
msgid "%s not found in archive - no files restored."
msgstr "%s wurde nicht im Archiv gefunden - keine Dateien wiederhergestellt."

#: ../duplicity/dup_main.py:866
msgid "No files found in archive - nothing restored."
msgstr "Keine Dateien im Archiv - nichts wiederhergestellt."

#: ../duplicity/dup_main.py:904
#, python-format
msgid "Processed volume %d of %d"
msgstr "Datenträger %d von %d wurde verarbeitet"

#: ../duplicity/dup_main.py:947
#, python-format
msgid "Invalid data - %s hash mismatch for file:"
msgstr "Ungültige Daten – %s-Hash stimmt nicht für die Datei:"

#: ../duplicity/dup_main.py:949
#, python-format
msgid "Calculated hash: %s"
msgstr "Erstellter Hash: %s"

#: ../duplicity/dup_main.py:950
#, python-format
msgid "Manifest hash: %s"
msgstr "Manifest-Hashcode: %s"

#: ../duplicity/dup_main.py:957 ../duplicity/util.py:110
#, python-format
msgid "IGNORED_ERROR: Warning: ignoring error as requested: %s: %s"
msgstr "VERNACHLÄSSIGTER FEHLER: Achtung: Fehler wird wie verlangt vernachlässigt: %s: %s"

#: ../duplicity/dup_main.py:1004
#, python-format
msgid "Volume was signed by key %s, not %s"
msgstr "Datenträger wurde von Schlüssel %s, anstatt von %s unterschrieben"

#: ../duplicity/dup_main.py:1036
#, python-format
msgid "Verify complete: %s, %s."
msgstr "Überprüfung vollständig: %s, %s."

#: ../duplicity/dup_main.py:1037
#, python-format
msgid "%d file(s) compared"
msgstr "%d Datei(en) verglichen"

#: ../duplicity/dup_main.py:1037
#, python-format
msgid "%d difference(s) found"
msgstr "%d Differenz(e) gefunden"

#: ../duplicity/dup_main.py:1056
msgid "No extraneous files found, nothing deleted in cleanup."
msgstr "Keine überflüssigen Dateien gefunden, es wurde nichts gelöscht."

#: ../duplicity/dup_main.py:1071
msgid "Found the following file(s) to delete:"
msgstr "Folgende Datei(en) zum Löschen gefunden:"

#: ../duplicity/dup_main.py:1075
msgid "Run duplicity again with the --force option to actually delete."
msgstr "Bitte duplicity noch einmal mit der »--force«-Option ausführen, um tatsächlich zu löschen."

#: ../duplicity/dup_main.py:1121
msgid "There are backup set(s) at time(s):"
msgstr "Es bestehen Sicherungssätze zu folgenden Zeitpunkten:"

#: ../duplicity/dup_main.py:1123
msgid "Which can't be deleted because newer sets depend on them."
msgstr "Welche nicht gelöscht werden können, weil neuere Sicherungssätze davon abhängen."

#: ../duplicity/dup_main.py:1130
msgid "Current active backup chain is older than specified time.  However, it will not be deleted.  To remove all your backups, manually purge the repository."
msgstr "Aktuell aktive Sicherungskette ist älter als der angegebene Zeitpunkt. Sie wird nicht gelöscht. Um alle Sicherungen zu entfernen, bitte die Paketquelle manuell löschen."

#: ../duplicity/dup_main.py:1148
msgid "No old backup sets found, nothing deleted."
msgstr "Keine alten Sicherungssätze gefunden, nichts gelöscht."

#: ../duplicity/dup_main.py:1151
msgid "Deleting backup chain(s) at time:"
msgstr "Lösche Sicherungskette(n) gleichzeitig:"

#: ../duplicity/dup_main.py:1160
#, python-format
msgid "Deleting any incremental signature chain rooted at %s"
msgstr "Jede schrittweise Signaturkette die bei %s verankert ist wird gelöscht"

#: ../duplicity/dup_main.py:1162
#, python-format
msgid "Deleting any incremental backup chain rooted at %s"
msgstr "Jede schrittweise Sicherungskette die bei %s verankert ist wird gelöscht"

#: ../duplicity/dup_main.py:1165
#, python-format
msgid "Deleting complete signature chain %s"
msgstr "Vollständige Signaturkette wird gelöscht: %s"

#: ../duplicity/dup_main.py:1167
#, python-format
msgid "Deleting complete backup chain %s"
msgstr "Vollständige Sicherungskette wird gelöscht: %s"

#: ../duplicity/dup_main.py:1174
msgid "Found old backup chain(s) at the following time:"
msgstr "Alte Sicherungskette(n) zur folgenden Zeit gefunden:"

#: ../duplicity/dup_main.py:1178
msgid "Rerun command with --force option to actually delete."
msgstr "Bitte Befehl erneut mit »--force«-Option ausführen, um tatsächlich zu löschen."

#: ../duplicity/dup_main.py:1281
#, python-format
msgid "Deleting local %s (not authoritative at backend)."
msgstr "Lokal %s wird gelöscht (keine Auswirkungen am Hintergrundprogramm)."

#: ../duplicity/dup_main.py:1285
#, python-format
msgid "Unable to delete %s: %s"
msgstr "%s kann nicht gelöscht werden: %s"

#: ../duplicity/dup_main.py:1319 ../duplicity/dup_temp.py:296
#, python-format
msgid "Failed to read %s: %s"
msgstr "%s konnte nicht gelesen werden: %s"

#: ../duplicity/dup_main.py:1331
#, python-format
msgid "Copying %s to local cache."
msgstr "%s wird zum lokalen Puffer kopiert."

#: ../duplicity/dup_main.py:1379
msgid "Local and Remote metadata are synchronized, no sync needed."
msgstr "Lokale und entfernte Metadaten sind bereits synchron, kein Abgleich benötigt."

#: ../duplicity/dup_main.py:1384
msgid "Synchronizing remote metadata to local cache..."
msgstr "Entfernte Metadaten werden zum lokalen Puffer synchronisiert …"

#: ../duplicity/dup_main.py:1398
msgid "Sync would copy the following from remote to local:"
msgstr "Synchronisation würde die folgenden Daten vom entfernten Rechner hierher kopieren:"

#: ../duplicity/dup_main.py:1404
msgid "Sync would remove the following spurious local files:"
msgstr "Synchronisation würde die folgenden überflüssigen lokalen Dateien entfernen:"

#: ../duplicity/dup_main.py:1450
msgid "Unable to get free space on temp."
msgstr "Der temporäre Speicherplatz ist belegt."

#: ../duplicity/dup_main.py:1457
msgid "Temp space has {freespace:,} available, backup needs approx {needspace:,}."
msgstr ""

#: ../duplicity/dup_main.py:1461
msgid "Temp has {freespace:,} available, backup will use approx {needspace:,}."
msgstr ""

#: ../duplicity/dup_main.py:1468
msgid "Unable to get max open files."
msgstr "Der Höchstwert für offene Dateien konnte nicht festgestellt werden."

#: ../duplicity/dup_main.py:1473
#, python-brace-format
msgid "Max open files of {maxopen} is too low, should be >= 1024.\n"
msgstr ""

#: ../duplicity/dup_main.py:1528
msgid "RESTART: The first volume failed to upload before termination.\n"
"         Restart is impossible...starting backup from beginning."
msgstr "NEUSTART: Das Hochladen des ersten Datenträgers ist fehlgeschlagen.\n"
"         Neustart nicht möglich, Sicherung wird vom Anfang wiederaufgenommen."

#: ../duplicity/dup_main.py:1538
#, python-format
msgid "RESTART: Volumes %d to %d failed to upload before termination.\n"
"         Restarting backup at volume %d."
msgstr "NEUSTART: Das Hochladen der Datenträger %d bis %d ist fehlgeschlagen.\n"
"        Sicherung wird mit Datenträger %d wiederaufgenommen."

#: ../duplicity/dup_main.py:1549
#, python-format
msgid "RESTART: Impossible backup state: manifest has %d vols, remote has %d vols.\n"
"         Restart is impossible ... duplicity will clean off the last partial\n"
"         backup then restart the backup from the beginning."
msgstr "NEUSTART: Unmöglicher Sicherungszustand:\n"
"Manifest hat eine Größe von %d, Hintergrundprogramm hat eine Größe von %d.\n"
"        Neustart nicht möglich … duplicity wird die letzte unvollständige\n"
"        Sicherung entfernen und von vorn anfangen."

#: ../duplicity/dup_main.py:1575
msgid "                PYTHONOPTIMIZE in the environment causes duplicity to fail to\n"
"                recognize its own backups.  Please remove PYTHONOPTIMIZE from\n"
"                the environment and rerun the backup.\n\n"
"                See https://bugs.launchpad.net/duplicity/+bug/931175\n"
"                "
msgstr ""

#: ../duplicity/dup_main.py:1655
#, python-brace-format
msgid "Last {action} backup left a partial set, restarting."
msgstr "Letzte {action} Sicherung hat einen Teilsatz hinterlassen, Neustart hinterlassen."

#: ../duplicity/dup_main.py:1659
#, python-brace-format
msgid "Cleaning up previous partial {action} backup set, restarting."
msgstr "Aufräumen der vorherigen partiellen {action} Sicherungsset, Neustart."

#: ../duplicity/dup_main.py:1673
msgid "Last full backup date: none"
msgstr "Letzte vollständige Sicherung: Keine"

#: ../duplicity/dup_main.py:1680
msgid "Last full backup is too old, forcing full backup"
msgstr "Letzte vollständige Sicherung ist zu alt, vollständige Sicherung wird erzwungen"

#: ../duplicity/dup_main.py:1736
msgid "When using symmetric encryption, the signing passphrase must equal the encryption passphrase."
msgstr "Bei symmetrischer Verschlüsselung müssen die Passphrasen für Signatur und Verschlüsselung übereinstimmen."

#: ../duplicity/dup_time.py:54
#, python-format
msgid "Bad interval string \"%s\"\n\n"
"Intervals are specified like 2Y (2 years) or 2h30m (2.5 hours).  The\n"
"allowed special characters are s, m, h, D, W, M, and Y.  See the man\n"
"page for more information."
msgstr "Ungültige Intervallzeichenkette »%s«\n\n"
"Intervalle sind angegeben wie 2Y (2 Jahre) oder 2h30m (2,5 Stunden).\n"
"Die erlaubten Spezialzeichen sind s, m, h, D, W, M, und Y.\n"
"Weitere Informationen sind im Handbuch zu finden."

#: ../duplicity/dup_time.py:63
#, python-format
msgid "Bad time string \"%s\"\n\n"
"The acceptible time strings are intervals (like \"3D64s\"), w3-datetime\n"
"strings, like \"2002-04-26T04:22:01-07:00\" (strings like\n"
"\"2002-04-26T04:22:01\" are also acceptable - duplicity will use the\n"
"current time zone), or ordinary dates like 2/4/1997 or 2001-04-23\n"
"(various combinations are acceptable, but the month always precedes\n"
"the day)."
msgstr "Ungültige Zeitzeichenkette »%s«\n\n"
"Gültige Zeitzeichenkette sind Intervalle (wie »3D46s«), w3-Datum-Zeitzeichenketten,\n"
"wie »2002-04-26T04:22:01-07:00« (Zeichenketten wie »2002-04-26T04:22:01« sind\n"
"auch gültig - duplicity wird die aktuelle Zeitzone nutzen, oder gewöhnliche Daten\n"
"wie 2/4/1997 oder 2001-04-23 (verschiedene Kombinationen sind gültig, aber vor\n"
"dem Monat kommt immer der Tag)."

#: ../duplicity/gpginterface.py:694
#, python-brace-format
msgid "GPG process {process.pid} terminated before wait()"
msgstr "GPG Prozess {process.pid} beendet vor wait()"

#: ../duplicity/lazy.py:349
#, python-format
msgid "Warning: oldindex %s >= newindex %s"
msgstr "Achtung: Alt-Index %s >= Neu-Index %s"

#: ../duplicity/lazy.py:424
#, python-format
msgid "Error '%s' processing %s"
msgstr "Fehler »%s« beim Verarbeiten von %s"

#: ../duplicity/lazy.py:436
#, python-format
msgid "Skipping %s because of previous error"
msgstr "%s wird wegen vorherigem Fehler übersprungen"

#: ../duplicity/librsync.py:194
msgid "basis_file must be a (true) file or an object whose file attribute is the underlying true file object"
msgstr "basis_file muss eine (echte) Datei oder ein Objekt sein, dessen Datei-Attribut das zugrundeliegende Dateiobjekt ist"

#: ../duplicity/manifest.py:95
#, python-format
msgid "Fatal Error: Backup source host has changed.\n"
"Current hostname: %s\n"
"Previous hostname: %s"
msgstr "Kritischer Fehler: Sicherungsquellrechner hat sich geändert.\n"
"Aktueller Rechnername: %s\n"
"Vorheriger Rechnername: %s"

#: ../duplicity/manifest.py:102
msgid "Fatal Error: Backup source directory has changed.\n"
msgstr "Schwerer Fehler: Backup-Quellverzeichnis wurde geändert.\n"

#: ../duplicity/manifest.py:115
msgid "Aborting because you may have accidentally tried to backup two different data sets to the same remote location, or using the same archive directory.  If this is not a mistake, use the --allow-source-mismatch switch to avoid seeing this message"
msgstr "Abbruch, weil Sie möglicherweise versehentlich versucht haben, zwei verschiedene Datensätze zum gleichen entfernten Ort oder zum selben Archivverzeichnis zu sichern. Wenn dies kein Fehler ist, verwenden Sie den »allow-source-mismatch«-Parameter, um diese Meldung auszuschalten."

#: ../duplicity/manifest.py:221
#, python-format
msgid "Found manifest volume %s"
msgstr "Manifest-Volumen %s gefunden"

#: ../duplicity/manifest.py:228
#, python-format
msgid "Found %s volumes in manifest"
msgstr "%s Volumen in Manifest gefunden"

#: ../duplicity/manifest.py:249
msgid "Manifest file '{self.fh.base if self.fh else ''}' is corrupt: "
msgstr "Manifest-Datei '{self.fh.base if self.fh else ''}' ist beschädigt: "

#: ../duplicity/manifest.py:268
msgid "Manifests not equal because different volume numbers"
msgstr "Manifests sind ungleich aufgrund verschiedener Datenträgerzahlen"

#: ../duplicity/manifest.py:273
msgid "Manifests not equal because volume lists differ"
msgstr "Manifests sind ungleich aufgrund verschiedenen Datenträgerlisten"

#: ../duplicity/manifest.py:277
msgid "Manifests not equal because hosts or directories differ"
msgstr "Manifests sind ungleich aufgrund Rechner- oder Verzeichnisunterschieden"

#: ../duplicity/manifest.py:438
msgid "Warning, found extra Volume identifier"
msgstr "Achtung, zusätzliche Datenträgerbezeichnung gefunden"

#: ../duplicity/manifest.py:464
msgid "Other is not VolumeInfo"
msgstr "Anderes ist keine Datenträgerinfo"

#: ../duplicity/manifest.py:467
msgid "Volume numbers don't match"
msgstr "Datenträgerzahlen stimmen nicht überein"

#: ../duplicity/manifest.py:470
msgid "start_indicies don't match"
msgstr "Startindizes stimmen nicht überein"

#: ../duplicity/manifest.py:473
msgid "end_index don't match"
msgstr "Endindex stimmt nicht überein"

#: ../duplicity/manifest.py:478
msgid "Hashes don't match"
msgstr "Prüfsummen stimmen nicht überein"

#: ../duplicity/patchdir.py:73 ../duplicity/patchdir.py:80
#, python-format
msgid "Patching %s"
msgstr "%s wird repariert"

#: ../duplicity/patchdir.py:548
#, python-format
msgid "Error '%s' patching %s"
msgstr "Fehler »%s« beim Reparieren von %s"

#: ../duplicity/patchdir.py:625
#, python-format
msgid "Writing %s of type %s"
msgstr "Schreibvorgang von %s des Typs %s"

#: ../duplicity/path.py:109
#, python-format
msgid "Warning: %s invalid devnums (0x%X), treating as (0, 0)."
msgstr "Achtung: %s ungültige Gerätenummern (0x%X), diese werden wie (0, 0) behandelt."

#: ../duplicity/path.py:239 ../duplicity/path.py:299
#, python-format
msgid "Warning: %s has negative mtime, treating as 0."
msgstr "Achtung: %s hat eine negative mtime, diese wird wie 0 behandelt."

#: ../duplicity/path.py:372
#, python-format
msgid "New file %s"
msgstr "Neue Datei %s"

#: ../duplicity/path.py:375
#, python-format
msgid "File %s is missing"
msgstr "Datei %s fehlt"

#: ../duplicity/path.py:378
#, python-format
msgid "File %%s has type %s, expected %s"
msgstr "Datei %%s ist vom Typ %s, erwartet wurde %s"

#: ../duplicity/path.py:383 ../duplicity/path.py:412
#, python-format
msgid "File %%s has permissions %s, expected %s"
msgstr "Datei %%s hat die Zugriffsrechte %s, erwartet wurde %s"

#: ../duplicity/path.py:389
#, python-format
msgid "File %%s has mtime %s, expected %s"
msgstr "Datei %%s hat mtime %s, erwartet wurde %s"

#: ../duplicity/path.py:400
#, python-format
msgid "Data for file %s is different"
msgstr "Daten für Datei %s sind unterschiedlich"

#: ../duplicity/path.py:408
#, python-format
msgid "Symlink %%s points to %s, expected %s"
msgstr "Symbolischer Verweis %%s zeigt auf %s, erwartet wurde %s"

#: ../duplicity/path.py:415
#, python-format
msgid "Device file %%s has numbers %s, expected %s"
msgstr "Gerätedatei %%s hat Nummern %s, erwartet wurde %s"

#: ../duplicity/path.py:605
#, python-format
msgid "Making directory %s"
msgstr "Verzeichnis %s wird erstellt"

#: ../duplicity/path.py:615
#, python-format
msgid "Deleting %s"
msgstr "»%s« wird gelöscht"

#: ../duplicity/path.py:624
#, python-format
msgid "Touching %s"
msgstr "Zugriffszeit %s wird erstellt oder aktualisiert"

#: ../duplicity/path.py:632
#, python-format
msgid "Deleting tree %s"
msgstr "Baumstruktur %s wird gelöscht"

#: ../duplicity/robust.py:84
#, python-format
msgid "Error listing directory %s"
msgstr "Fehler beim Auflisten des Verzeichnisses %s"

#: ../duplicity/selection.py:115
#, python-format
msgid "Skipping socket %s"
msgstr "Socket %s wird übersprungen"

#: ../duplicity/selection.py:121
#, python-format
msgid "Error initializing file %s"
msgstr "Fehler beim Initialisieren der Datei %s"

#: ../duplicity/selection.py:127 ../duplicity/selection.py:167
#: ../duplicity/selection.py:615
#, python-format
msgid "Error accessing possibly locked file %s"
msgstr "Fehler beim Zugriff auf möglicherweise gesperrte Datei %s"

#: ../duplicity/selection.py:182
#, python-format
msgid "Warning: base %s doesn't exist, continuing"
msgstr "Achtung: Basis %s besteht nicht, es wird fortgefahren"

#: ../duplicity/selection.py:184 ../duplicity/selection.py:205
#: ../duplicity/selection.py:208
#, python-format
msgid "Selecting %s"
msgstr "%s wird ausgewählt"

#: ../duplicity/selection.py:263
#, python-format
msgid "                The last file selection option is the filter option %s, which will have no\n"
"                effect as there are no subsequent file selection options. Exiting because\n"
"                this probably isn't what you meant."
msgstr "                Die letzte Dateiauswahloption ist die Filteroption %s, die keinen\n"
"                Effekt haben, da es keine nachfolgenden Dateiauswahloptionen gibt. Verlasse weil\n"
"                wahrscheinlich nicht das ist, was du meinst."

#: ../duplicity/selection.py:278
#, python-format
msgid "                Only these filter mode options were specified:\n"
"                    %s\n"
"                Case sensitive globbing is the default behaviour and so this has no effect.\n"
"                Exiting because this probably isn't what you meant."
msgstr "                Es wurden nur diese Filtermodus-Optionen angegeben:\n"
"                    %s\n"
"                Groß-/Kleinschreibung ist das Standardverhalten und dies hat keine Wirkung.\n"
"                Verweilen, weil dies wahrscheinlich nicht das ist, was du meinst."

#: ../duplicity/selection.py:346
#, python-format
msgid "                Fatal Error: The file specification\n"
"                    %s\n"
"                cannot match any files in the base directory\n"
"                    %s\n"
"                Useful file specifications begin with the base directory or some\n"
"                pattern (such as '**') which matches the base directory."
msgstr "                Schwerer Fehler: Die Datei-Spezifikation\n"
"                    %s\n"
"                kann keine Dateien im Basisverzeichnis\n"
"                    %s\n"
"                Nützliche Datei-Spezifikationen beginnen mit dem Basisverzeichnis oder einem\n"
"                Muster (wie '**'), das dem Basisverzeichnis entspricht."

#: ../duplicity/selection.py:360
#, python-format
msgid "Fatal Error while processing expression\n"
"%s"
msgstr "Kritischer Fehler beim Verarbeiten des Ausdrucks\n"
"%s"

#: ../duplicity/selection.py:378
#, python-format
msgid "Reading files to backup from list %s"
msgstr "Lese Dateien zur Sicherung aus der Liste %s"

#: ../duplicity/selection.py:403
#, python-format
msgid "                Files-from list contains the absolute path:\n"
"                    %s\n"
"                All paths specified in a files-from list must be given relative to the backup\n"
"                source path."
msgstr "                Dateien aus der Liste enthalten den absoluten Pfad:\n"
"                    %s\n"
"                Alle Pfade, die in einer Liste von Dateien angegeben sind, müssen relativ zum Quellpfad\n"
"                angegeben werden."

#: ../duplicity/selection.py:418
msgid "                Files-from list specified which contains no files, the backup will be empty as\n"
"                a result. Exiting as this probably isn't what you meant,"
msgstr "                Dateien aus der Liste angegeben, die keine Dateien enthält, wird die Sicherung als\n"
"                ein Ergebnis leer sein. Verlassen Sie dies, da dies wahrscheinlich nicht das ist, was Sie meinten,"

#: ../duplicity/selection.py:435
#, python-format
msgid "                Last selection expression:\n"
"                    %s\n"
"                only specifies that files be included.  Because the default is to\n"
"                include all files, the expression is redundant.  Exiting because this\n"
"                probably isn't what you meant."
msgstr "                Letzter Ausdruck:\n"
"                    %s\n"
"                gibt nur an, dass Dateien eingebunden werden. Weil die Standardeinstellung\n"
"                alle Dateien enthalten ist der Ausdruck ist überflüssig. Verlassen, weil dieses\n"
"                wahrscheinlich nicht das ist, was Sie meinten."

#: ../duplicity/selection.py:497
#, python-format
msgid "Reading %s filelist %s"
msgstr "Lese %s Dateiliste %s"

#: ../duplicity/selection.py:538
#, python-format
msgid "Error compiling regular expression %s"
msgstr "Fehler beim Auswerten des regulären Ausdrucks %s"

#: ../duplicity/tempdir.py:157
#, python-format
msgid "Using temporary directory %s"
msgstr "Temporäres Verzeichnis %s wird benutzt"

#: ../duplicity/tempdir.py:201
#, python-format
msgid "Registering (mktemp) temporary file %s"
msgstr "(mktemp) temporäre Datei %s wird registriert"

#: ../duplicity/tempdir.py:227
#, python-format
msgid "Registering (mkstemp) temporary file %s"
msgstr "(mkstemp) temporäre Datei %s wird registriert"

#: ../duplicity/tempdir.py:259
#, python-format
msgid "Forgetting temporary file %s"
msgstr "Temporäre Datei %s wird vergessen"

#: ../duplicity/tempdir.py:262
#, python-format
msgid "Attempt to forget unknown tempfile %s - this is probably a bug."
msgstr "Unbekannte temporäre Datei %s wird versucht zu vergessen - das ist wahrscheinlich ein Fehler."

#: ../duplicity/tempdir.py:281
#, python-format
msgid "Removing still remembered temporary file %s"
msgstr "Bereits gemerkte temporäre Datei %s wird entfernt"

#: ../duplicity/tempdir.py:284
#, python-format
msgid "Cleanup of temporary file %s failed"
msgstr "Entfernen der temporären Datei %s ist fehlgeschlagen"

#: ../duplicity/tempdir.py:290
#, python-format
msgid "Cleanup of temporary directory %s failed - this is probably a bug."
msgstr "Aufräumen des temporären Verzeichnisses %s fehlgeschlagen - das ist wahrscheinlich ein Fehler."

#: ../duplicity/util.py:176
#, python-format
msgid "Acquiring lockfile %s"
msgstr "Die Sperrdatei %s wird erfasst"

#: ../duplicity/util.py:191
#, python-format
msgid "Releasing lockfile %s"
msgstr "Sperrdatei »%s« wird freigegeben"

